integration/spark/app/build.gradle

```diff
@@ -56,10 +56,10 @@ ext {
     testcontainersVersion = '1.15.3'
     shortVersion = sparkVersion.substring(0,3)
     versionsMap = [
-            "3.3": ["module": "spark33",    "scala": "2.12"],
-            "3.2": ["module": "spark32",    "scala": "2.12"],
-            "3.1": ["module": "spark3",     "scala": "2.12"],
-            "2.4": ["module": "spark2",     "scala": "2.11"]
+            "3.3": ["module": "spark33",    "scala": "2.12",    "jackson": "2.10.0"],
+            "3.2": ["module": "spark32",    "scala": "2.12",    "jackson": "2.10.0"],
+            "3.1": ["module": "spark3",     "scala": "2.12",    "jackson": "2.10.0"],
+            "2.4": ["module": "spark2",     "scala": "2.11",    "jackson": "2.6.7"]
```

Comment:
Just because we recently had issues with Jackson being installed in the OL Jar (https://github.com/OpenLineage/OpenLineage/issues/852) what does the inclusion of jackson here do?

Reply:
those dependencies are needed in test scope, but I'm still searching for some better solution